src/main/webapp/app/app.tsx

```diff
@@ -47,11 +47,13 @@ export class App extends React.Component<IAppProps> {
               isSwaggerEnabled={this.props.isSwaggerEnabled}
             />
           </ErrorBoundary>
-          <div className="flex-grow-1">
-            <ErrorBoundary>
-              <AppRoutes />
-            </ErrorBoundary>
-          </div>
+          <Container>
```

Comment:
what purpose for this?

Reply:
@junzai97 
i think i knw the purpose of this
because you want to avoid the page content stick at the border of the screen
but maybe you need to change to apply it inside the div

@kachun333 i believe you remove the Container tag when you doing the auth module right?

Reply:
@HowardYaw 
yes, you are right. I removed the <Container> tag when I did the auth module